serdect/src/array.rs

```diff
@@ -1,11 +1,20 @@
 //! Serialization primitives for arrays.
 
-use core::fmt;
+// Unfortunately, we currently cannot assert generically that we are serializing
+// a fixed-size byte array.
```

Comment:
I'm not sure what this sentence says, do you mean to say that we can't know if the tuple type of a serialization format fits our needs?

Reply:
Perhaps it is not very well phrased. By "generically" I meant generically over formats supported by `serde`, not generically in the sense of Rust type system generics.

Reply:
Amended the phrasing.